### PR TITLE
Meta: encourage tracking of dependent specifications

### PR DESCRIPTION
**Pull request by domenic**

### CODE REVIEW

File: infra.bs [hunk 1]
```diff
@@ -47,9 +47,14 @@ urlPrefix: https://heycam.github.io/webidl/; spec: WEBIDL; type: dfn
 <h2 id=usage>Usage</h2>
 
 <p>To make use of the Infra Standard in a document titled <var>X</var>, use
-<samp><var>X</var> depends on the Infra Standard.</samp> Additionally, cross-referencing terminology
+<samp><var>X</var> depends on the Infra Standard</samp>. Additionally, cross-referencing terminology
```
Comment (annevk) [inline infra.bs]: I placed the dot inside since it's part of the text. But I guess this is okay.

File: infra.bs [hunk 2]
```diff
@@ -47,9 +47,14 @@ urlPrefix: https://heycam.github.io/webidl/; spec: WEBIDL; type: dfn
...
 is encouraged to avoid ambiguity.
 
+<p>Specification authors are also encouraged to add their specification to the
+<a href=https://github.com/whatwg/infra/wiki/Dependent-specs>list of dependent specifications</a> in
+order to help the editors ensure that any future changes to the Infra Standard are correctly
```
Comment (annevk) [inline infra.bs]: s/future/incompatible/?

Comment (domenic) [inline infra.bs]: Maybe "future breaking changes"

Comment (annevk) [inline infra.bs]: LGTM with that.

**Pull request merged**